documentation/asciidoc/topics/ref_xsite_crd.adoc

```diff
@@ -41,7 +27,14 @@ This secret contains different authentication objects, depending on your
 Kubernetes environment.
 ====
 endif::productized[]
+
 //Product
 ifdef::productized[]
-<6> specifies the secret that contains the service account token for the backup site.
+<1> specifies the DataGrid service. {brandname} supports cross-site replication with the DataGrid service only.
+<2> names the local site for a {brandname} cluster.
+<3> specifies `LoadBalancer` as the service that exposes the {openshiftshort} cluster to externally network traffic.
```

Comment:
The `LoadBalancer` here is used to communicate between the two sites. It is not for handling external traffic.